bciers/apps/reporting/src/tests/components/signOff/SignOffForm.test.tsx

```diff
@@ -24,65 +64,54 @@ describe("SignOffForm Component", () => {
     });
   });
 
-  it("renders the form with correct fields and values", async () => {
+  it("renders the form with the correct text and fields", async () => {
     renderSignOffForm();
-    expect(
-      screen.getByText(
-        "I certify that I have reviewed the annual report, and that I have exercised due diligence to ensure that the information included in this report is true and complete.",
-      ),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByText(
-        "I understand that the Ministry responsible for the administration and enforcement of the Greenhouse Gas Industrial Reporting and Control Act may require records from the Operator evidencing the truth of this report.",
-      ),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByText(
-        "I understand that this information is being collected for the purpose of emission reporting under the Greenhouse Gas Industrial Reporting and Control Act and may be disclosed to the Ministry responsible for the administration and enforcement of the Carbon Tax Act.",
-      ),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByText(
-        "I understand that the information provided in this report will impact the compliance obligation of this operation and that any errors, omissions, or misstatements can lead to an additional compliance obligation or administrative penalties.",
-      ),
-    ).toBeInTheDocument();
+
+    // Check for the presence of the important texts
+    const texts = [
+      "I certify that I have reviewed the annual report, and that I have exercised due diligence to ensure that the information included in this report is true and complete.",
+      "I understand that the Ministry responsible for the administration and enforcement of the Greenhouse Gas Industrial Reporting and Control Act may require records from the Operator evidencing the truth of this report.",
+      "I understand that this information is being collected for the purpose of emission reporting under the Greenhouse Gas Industrial Reporting and Control Act and may be disclosed to the Ministry responsible for the administration and enforcement of the Carbon Tax Act.",
+      "I understand that the information provided in this report will impact the compliance obligation of this operation and that any errors, omissions, or misstatements can lead to an additional compliance obligation or administrative penalties.",
+    ];
+
+    texts.forEach((text) => {
+      expect(screen.getByText(text)).toBeInTheDocument();
+    });
```

Comment:
This test is now tautological - we mock something and test the thing we mocked. 

It's now not testing the actual schema contents anymore. The schema generation method introduced in this PR needs to be tested at the very least, or refactored to make use of multiple schemas (see previous comment), so we can trust that it gets rendered on screen